scripts/variantstore/wdl/extract/summarize_task_monitor_logs.py

```diff
@@ -0,0 +1,192 @@
+import argparse
+import sys
+import re
+import os
+
+def parse_monitoring_log_files(mlog_files, output_file):
+    with open(output_file, 'w') as output:
+        header = f"Total Mem\tMax Mem Used\tMax Mem Used (%)\tTotal Disk\tMax Disk Used\tMax Disk Used (" \
+                 f"%)\tTask\tShard\tFile\n"
+        output.write(header)
+
+        for mlog_file in mlog_files:
+            if (not os.path.exists(mlog_file)):
+                eprint(f"ERROR: File {mlog_file} does not exist")
+            parse_monitoring_log_file(mlog_file, output)
+
+
+def parse_monitoring_log_file(mlog_file, output):
+    eprint(f"Parsing: {mlog_file}")
+
+    if (os.stat(mlog_file).st_size == 0):
```

Comment:
Yay for the PEP 8 fixups, but I'm still seeing a lot of non-PEP 8 warnings in IntelliJ. e.g. on this line "Remove redundant parantheses". If/when we go to our own repo there will likely be Python linting that will error on issues like this. Happy to review in mobbing to make sure we're seeing the same thing!